core-guides/getting-started.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Getting started"
+---
+
+As mentioned before, this is the place to start if you are interested in participating the Core Datasets project.
+
+The first and immediate thing you should do is to sign-up in the [forum](https://discuss.okfn.org/) and introduce yourself.
+The forums act as our central place of communications. However, we also discuss a lot through GitHub, since we keep track 
+of our project's management through the [issues tab](https://github.com/datasets/registry/issues).
+
+[This video](https://vimeo.com/133587259) will help you understand the core functionalities of the platform where we host 
+the forums. Worry not!, you will find most of them very intuitive and will learn along the way. In fact, feel free to ask for 
+help in there if you feel lost.
+
+In order to keep things organized in GitHub, we use the label system. They are very straightforward and easy to understand. 
+It is important that you respect those, that are usually attributed by Core Datasets Managers. This group of people keeps 
+track of the project's needs and tries to forward the project's direction accordingly.
+
+Following the project's direction
```

Comment:
For markdown we want to use `##  ` method for doing headings rather than underling.